manifests/overlays/moc-infra/applications/kustomization.yaml

```diff
@@ -1,6 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - app-of-apps-curator.yaml
-  - app-of-apps-infra.yaml
-  - app-of-apps-zero.yaml
+  - github.com/operate-first/argocd-apps/app-of-apps?ref=master
```

Comment:
FTR, added to the list here https://github.com/operate-first/blueprint/issues/50

Reply:
it should actually be `main` woops